### DISCUSSION TITLE
q-table threshold not working

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#9530] [CLI - PWA mode]
Hello All

i have an issue with my logic when i input my value. am not able to spot the issue
Can some please help

https://codepen.io/akashramtohul4/pen/poeaRgL

**Comment by ontwikkelfabriek** [comment]
@akashramtohul4 

 I know nobody replied to your request for help in your other post, but please `don't post the same thing twice`. If you're still stuck after a few days you could bump up your post by replying to your own post(instead of creating a new topic).

 I took a look at it and it's quite a puzzle....

**Comment by nucle** [comment]
Hi,

this simple check is only for ThresholdStart. 
I would do something like that:
```javascript
ThresholdRangeValidation(selectedRow)
{
    if (!selectedRow) {
        return
    }
    const withoutSelectedRow = this.data.filter(row => row.id != selectedRow.id)
    const foundInvalidThreshold = withoutSelectedRow.filter(row => {
        // Other checks ...
        if (selectedRow.ThresholdStart >= row.ThresholdStart && selectedRow.ThresholdStart < row.ThresholdEnd) {
            return row
        }
    })

    const isValid = foundInvalidThreshold.length <= 0
    
    // Here your error message

    return isValid
}
```
For simplicity, I would use two different methods for validation.
ThresholdStartRangeValidation and ThresholdEndRangeValidation
Later you can merge and combine everything.

When everything works you can combine the filters to one.
BR
